inst/metadata/advanced_run_options.json

```diff
@@ -199,8 +169,26 @@
     {
       "label": "t_(OPTIONS_ADVANCED_DEFF_PROPORTION_RECENT_LABEL)",
       "controls": [
+          "name": "use_kish_recent",
```

Comment:
Missing an `{` here.

You can always paste this json into a validator. Either something like https://jsonlint.com/ or in R itself `jsonlite::read_json("path/to/file")` note that things which use the templating will fail though as template is not valid json. Using an online tool should help distinguish between actual errors and just ones from templating

Reply:
Thanks, fixed in 73e91ce.